From c1e98f6924b2f8c35544f2dda5e92d4e86dd9825 Mon Sep 17 00:00:00 2001
From: root <c@c.com>
Date: Wed, 29 Apr 2020 18:43:57 +0000
Subject: [PATCH] change port to 7777

---
 app.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app.py b/app.py
index d3c9203..1efd75a 100644
--- a/app.py
+++ b/app.py
@@ -11,4 +11,4 @@ def hello_world():
 
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+    app.run(host='0.0.0.0', port=7777)
-- 
2.7.4

